{
	"id": "RLSA-2020:5620",
	"advisories": [
		{
			"content": {
				"id": "RLSA-2020:5620",
				"title": "Important: postgresql:12 security update",
				"description": "PostgreSQL is an advanced object-relational database management system (DBMS).\n\nThe following packages have been upgraded to a later upstream version: postgresql (12.5).\n\nSecurity Fix(es):\n\n* postgresql: Reconnection can downgrade connection security settings (CVE-2020-25694)\n\n* postgresql: Multiple features escape \"security restricted operation\" sandbox (CVE-2020-25695)\n\n* postgresql: Uncontrolled search path element in logical replication (CVE-2020-14349)\n\n* postgresql: Uncontrolled search path element in CREATE EXTENSION (CVE-2020-14350)\n\n* postgresql: psql's \\gset allows overwriting specially treated variables (CVE-2020-25696)\n\n* postgresql: ALTER ... DEPENDS ON EXTENSION is missing authorization checks (CVE-2020-1720)\n\nFor more details about the security issue(s), including the impact, a CVSS score, acknowledgments, and other related information, refer to the CVE page(s) listed in the References section.",
				"severity": [
					{
						"type": "vendor",
						"source": "errata.rockylinux.org",
						"vendor": "IMPORTANT"
					}
				],
				"references": [
					{
						"source": "MITRE",
						"url": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-14349"
					},
					{
						"source": "MITRE",
						"url": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-14350"
					},
					{
						"source": "MITRE",
						"url": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-1720"
					},
					{
						"source": "MITRE",
						"url": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-25694"
					},
					{
						"source": "MITRE",
						"url": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-25695"
					},
					{
						"source": "MITRE",
						"url": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-25696"
					},
					{
						"source": "Red Hat",
						"url": "https://bugzilla.redhat.com/show_bug.cgi?id=1798852"
					},
					{
						"source": "Red Hat",
						"url": "https://bugzilla.redhat.com/show_bug.cgi?id=1865744"
					},
					{
						"source": "Red Hat",
						"url": "https://bugzilla.redhat.com/show_bug.cgi?id=1865746"
					},
					{
						"source": "Red Hat",
						"url": "https://bugzilla.redhat.com/show_bug.cgi?id=1894423"
					},
					{
						"source": "Red Hat",
						"url": "https://bugzilla.redhat.com/show_bug.cgi?id=1894425"
					},
					{
						"source": "Red Hat",
						"url": "https://bugzilla.redhat.com/show_bug.cgi?id=1894430"
					},
					{
						"source": "errata.rockylinux.org",
						"url": "https://errata.rockylinux.org/RLSA-2020:5620"
					}
				],
				"published": "2020-12-17T15:30:10Z"
			},
			"segments": [
				{
					"ecosystem": "rocky:8"
				}
			]
		}
	],
	"vulnerabilities": [
		{
			"content": {
				"id": "CVE-2020-14349",
				"severity": [
					{
						"type": "cvss_v31",
						"source": "MITRE",
						"cvss_v31": {
							"vector": "CVSS:3.1/AV:N/AC:H/PR:L/UI:R/S:U/C:H/I:H/A:H",
							"base_score": 7.1,
							"base_severity": "HIGH",
							"temporal_score": 7.1,
							"temporal_severity": "HIGH",
							"environmental_score": 7.1,
							"environmental_severity": "HIGH"
						}
					}
				],
				"cwe": [
					{
						"source": "MITRE",
						"cwe": [
							"CWE-20"
						]
					}
				],
				"references": [
					{
						"source": "MITRE",
						"url": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-14349"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "rocky:8"
				}
			]
		},
		{
			"content": {
				"id": "CVE-2020-14350",
				"severity": [
					{
						"type": "cvss_v31",
						"source": "MITRE",
						"cvss_v31": {
							"vector": "CVSS:3.1/AV:N/AC:H/PR:L/UI:R/S:U/C:H/I:H/A:H",
							"base_score": 7.1,
							"base_severity": "HIGH",
							"temporal_score": 7.1,
							"temporal_severity": "HIGH",
							"environmental_score": 7.1,
							"environmental_severity": "HIGH"
						}
					}
				],
				"cwe": [
					{
						"source": "MITRE",
						"cwe": [
							"CWE-20"
						]
					}
				],
				"references": [
					{
						"source": "MITRE",
						"url": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-14350"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "rocky:8"
				}
			]
		},
		{
			"content": {
				"id": "CVE-2020-1720",
				"severity": [
					{
						"type": "cvss_v31",
						"source": "MITRE",
						"cvss_v31": {
							"vector": "CVSS:3.1/AV:N/AC:H/PR:L/UI:N/S:U/C:N/I:L/A:N",
							"base_score": 3.1,
							"base_severity": "LOW",
							"temporal_score": 3.1,
							"temporal_severity": "LOW",
							"environmental_score": 3.1,
							"environmental_severity": "LOW"
						}
					}
				],
				"cwe": [
					{
						"source": "MITRE",
						"cwe": [
							"CWE-285"
						]
					}
				],
				"references": [
					{
						"source": "MITRE",
						"url": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-1720"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "rocky:8"
				}
			]
		},
		{
			"content": {
				"id": "CVE-2020-25694",
				"severity": [
					{
						"type": "cvss_v31",
						"source": "MITRE",
						"cvss_v31": {
							"vector": "CVSS:3.1/AV:N/AC:H/PR:N/UI:N/S:U/C:H/I:H/A:H",
							"base_score": 8.1,
							"base_severity": "HIGH",
							"temporal_score": 8.1,
							"temporal_severity": "HIGH",
							"environmental_score": 8.1,
							"environmental_severity": "HIGH"
						}
					}
				],
				"cwe": [
					{
						"source": "MITRE",
						"cwe": [
							"CWE-327"
						]
					}
				],
				"references": [
					{
						"source": "MITRE",
						"url": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-25694"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "rocky:8"
				}
			]
		},
		{
			"content": {
				"id": "CVE-2020-25695",
				"severity": [
					{
						"type": "cvss_v31",
						"source": "MITRE",
						"cvss_v31": {
							"vector": "CVSS:3.1/AV:N/AC:L/PR:L/UI:N/S:U/C:H/I:H/A:H",
							"base_score": 8.8,
							"base_severity": "HIGH",
							"temporal_score": 8.8,
							"temporal_severity": "HIGH",
							"environmental_score": 8.8,
							"environmental_severity": "HIGH"
						}
					}
				],
				"cwe": [
					{
						"source": "MITRE",
						"cwe": [
							"CWE-89"
						]
					}
				],
				"references": [
					{
						"source": "MITRE",
						"url": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-25695"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "rocky:8"
				}
			]
		},
		{
			"content": {
				"id": "CVE-2020-25696",
				"severity": [
					{
						"type": "cvss_v31",
						"source": "MITRE",
						"cvss_v31": {
							"vector": "CVSS:3.1/AV:N/AC:H/PR:N/UI:R/S:U/C:H/I:H/A:H",
							"base_score": 7.5,
							"base_severity": "HIGH",
							"temporal_score": 7.5,
							"temporal_severity": "HIGH",
							"environmental_score": 7.5,
							"environmental_severity": "HIGH"
						}
					}
				],
				"cwe": [
					{
						"source": "MITRE",
						"cwe": [
							"CWE-183->CWE-270"
						]
					}
				],
				"references": [
					{
						"source": "MITRE",
						"url": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-25696"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "rocky:8"
				}
			]
		}
	],
	"detections": [
		{
			"ecosystem": "rocky:8",
			"conditions": [
				{
					"criteria": {
						"operator": "OR"
					}
				}
			]
		}
	],
	"data_source": {
		"id": "rocky-errata",
		"raws": [
			"vuls-data-raw-rocky-errata/2020/RLSA-2020:5620.json"
		]
	}
}
